packages/components/src/tabs/styles.ts

```diff
@@ -175,23 +195,69 @@ export const Tab = styled( Ariakit.Tab )`
 			opacity: 0;
 
 			@media not ( prefers-reduced-motion ) {
-				transition: opacity 0.1s linear;
+				transition: opacity 0.15s 0.15s linear;
```

Comment:
Just to be sure, is this intentional? This means the duration is 0.15s and the delay is 0.15s. The button component has a duration of 0.1s and no delay, which feels a bit unnatural to me:

https://github.com/user-attachments/assets/dd4f935a-dda9-40eb-8291-9ba5f20da819

Reply:
It is: https://github.com/WordPress/gutenberg/pull/65387#discussion_r1788055751

I believe idea is to show the chevron only after the "indicator" has animated to the new selected tab

Reply:
I saw the comment, but isn't that animation only expected for vertical tabs?

Additionally, I don't know if it's related to this PR, but I can see that the pattern categories in the block sidebar don't seem to animate properly - they don't have a background color when focused:


https://github.com/user-attachments/assets/f2ba1bc3-06ad-4bb2-89b9-926ca03f4235

Reply:
> but isn't that animation only expected for vertical tabs?

This seem like a good point. The delay (and maybe the whole transition) should only be needed for `data-select-on-move='true'` conditions, I think.

Reply:
On my phone, and on sick leave, but just wanted to say that as far as I remember, this transition was only applied to the Chevron in vertical tabs, and only when select on move is enabled and when using the arrow keys. If that's not the case, something got messed up, and that'd be surprising to me.

Reply:
See https://github.com/WordPress/gutenberg/pull/65387#discussion_r1797694534 😅 

Will fix soon. Silly me!

Reply:
Fix in https://github.com/WordPress/gutenberg/pull/66198 — it should also fix the pattern categories in the block sidebar mentioned by @t-hamano above.